test/integration/forms/form.js

```diff
@@ -350,6 +350,65 @@ context('Patient Action Form', function() {
       .should('have.value', 'bar');
   });
 
+  specify('prefill a form with latest submission by flow', function() {
+    cy
+      .routeAction(fx => {
+        fx.data.id = '1';
+        fx.data.relationships.form.data = { id: '11111' };
+        fx.data.relationships['program-action'] = { data: { id: '11111' } };
+        fx.data.relationships.flow = { data: { id: '1' } };
+
+        fx.data.attributes.tags = ['prefill-flow-response'];
+
+        return fx;
+      })
+      .routeForm(_.identity, '11111')
+      .routeFormDefinition()
+      .routeFormActionFields()
+      .routeActionActivity()
+      .routePatientByAction()
+      .routeLatestFormResponseByPatient(fx => {
+        fx.data.attributes = {
+          response: {
+            data: {
+              familyHistory: 'Prefilled family history',
+              storyTime: 'Prefilled story time',
+              patient: { fields: { foo: 'bar' } },
+            },
+          },
+        };
+
+        return fx;
+      })
+      .visit('/patient-action/1/form/11111')
+      .wait('@routeAction')
+      .wait('@routeForm')
+      .wait('@routePatientByAction')
+      .wait('@routeFormDefinition');
+
+    cy
+      .wait('@routeLatestFormResponseByPatient')
+      .itsUrl()
+      .its('search')
+      .should('contain', 'filter[form]=11111')
+      .should('contain', 'filter[flow]=1');
+
+    cy
+      .iframe()
+      .find('textarea[name="data[familyHistory]"]')
+      .should('have.value', 'Prefilled family history');
+
+    cy
+      .iframe()
+      .find('textarea[name="data[storyTime]"]')
+      .should('have.value', 'Prefilled story time');
+
+    cy
+      .iframe()
+      .find('[name="data[patient.fields.foo]"]')
+      .should('have.value', 'bar');
+  });
```

Comment:
Same as the existing `'prefill a form with latest submission from another form'` test, but with the ` ['prefill-flow-response']` tag and an added flow relationship: